EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -232,13 +250,77 @@ EarthAsDMAnalyzer::EarthAsDMAnalyzer(const edm::ParameterSet& iConfig) :
  tracksToken_(consumes<std::vector<reco::Track>>(iConfig.getParameter<edm::InputTag>("trackCollection")))
 {}
 
-
 EarthAsDMAnalyzer::~EarthAsDMAnalyzer()  = default;
 
 //
 // member functions
 //
 
+  using namespace std;
```

Comment:
what uses `std`? the `cerr`? if you have it as you do, you dont need to add the namespace